readthedocs/projects/forms.py

```diff
@@ -685,18 +687,8 @@ def __init__(self, *args, **kwargs):
         else:
             self.fields.pop("force")
 
-    def save(self, **_):  # pylint: disable=arguments-differ
-        # TODO this should respect the unused argument `commit`. It's not clear
-        # why this needs to be a call to `create`, instead of relying on the
-        # super `save()` call.
-        redirect = Redirect.objects.create(
-            project=self.project,
-            redirect_type=self.cleaned_data["redirect_type"],
-            from_url=self.cleaned_data["from_url"],
-            to_url=self.cleaned_data["to_url"],
-            force=self.cleaned_data.get("force", False),
-        )
-        return redirect
```

Comment:
We were doing this override to allow passing the project, but by using the trick of listing the project field and always using the project we pass to form this isn't needed anymore.